src/capitalsFirst.js

```diff
@@ -21,7 +21,18 @@
  * @returns {string} - string with uppercase words in front
  */
 function capitalsFirst(str) {
-  // write code here
+  let sortWords = [];
+  str.split(' ').forEach(word => {
+    if (word[0].charCodeAt() > 64 && word[0].charCodeAt() < 91) {
```

Comment:
Move magic numbers to constants with UPPER_CASE names outside of the function
Also you can create helper function isUppercaseLetter/isUppercaseWord